### PR TITLE
 Basic Add Account Logic

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7100]

File: src/api/api.ts [hunk 1]
```diff
@@ -36,15 +37,18 @@ const requestSigning = (
...
       return requestSigning(request, webContents);
-    // @todo Actual account handling
+    // @todo Dont expose all accounts at once?
+    // @todo Decide whether to fetch directly from store?
     case SUPPORTED_METHODS.ACCOUNTS:
       return toJsonRpcResponse({
         id: request.id,
-        result: ['0x82D69476357A03415E92B5780C89e5E9e972Ce75']
+        result: Object.values(await getAccounts())
+          .filter((a) => a.address)
```
Comment (Mrtenz) [inline src/api/api.ts:50]: Is this necessary? In what situation would `address` be `undefined` or `null`?

File: src/api/crypto.spec.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import { IPC_CHANNELS } from '@config';
+import { CryptoRequestType } from '@types';
+
+import { handleRequest, runService } from './crypto';
+
+jest.mock('@ethersproject/wallet', () => ({
+  Wallet: jest.fn().mockImplementation(() => ({
+    address: '0x4bbeEB066eD09B7AEd07bF39EEe0460DFa261520',
+    signTransaction: jest.fn().mockImplementation(() => Promise.resolve('signed'))
```
Comment (Mrtenz) [inline src/api/crypto.spec.ts:11]: Can we return an actually signed transaction here? That may prevent situations in the future where some library depends on a proper signed transaction hexadecimal string, but gets "signed" instead, something that broke a bunch of stuff in legacy.

File: src/app/screens/AddAccount.test.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+import { getAddressFromPrivateKey } from '@app/services/WalletService';
+import { ApplicationState, createStore } from '@app/store';
+
+import { AddAccount } from '.';
+
+jest.mock('@app/services/WalletService', () => ({
+  getAddressFromPrivateKey: jest
+    .fn()
+    .mockImplementation(() => Promise.resolve({ uuid: 'uuid', address: 'address' }))
```
Comment (Mrtenz) [inline src/app/screens/AddAccount.test.tsx:16]: Return actual UUID/address. See comment above.

File: src/types/route.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export interface Route {
+  path: string;
+  component: React.ComponentType<any>;
```
Comment (Mrtenz) [inline src/types/route.ts:3]: `any` -> `unknown`

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/api/api.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/api/api.ts:50]: It seems that some metadata is stored by `persist-redux`, maybe I should filter it out some other way. Hmm

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**